generate-web/error/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Registration error</title>
+        <link rel="stylesheet" href="../style.css">
+    </head>
+    <body>
+        <h1>Oops!</h1>
+        <p>Something went wrong with your email verification. Please go to <a href="../">http://quickstart.holo.host</a> and start the registration process again.</p>
```

Comment:
I don't understand why email verification is part of scope of this app. It's purely for generating config, email verification is orthogonal and handled later on.

Reply:
moving out to hpstatus repo